source/features/easy-toggle-files.tsx

```diff
@@ -15,14 +15,43 @@ function toggleFile(event: DelegateEvent<MouseEvent>): void {
 	}
 }
 
+function toggleCSFile(event: DelegateEvent<MouseEvent>): void {
+	const elementClicked = event.target as HTMLElement;
+	const headerBar = event.delegateTarget;
+
+	// The clicked element is either the bar itself or one of its children excluding the button
+	if (elementClicked === headerBar || (elementClicked.parentElement === headerBar && elementClicked.tagName !== 'BUTTON')) {
+		$(['[aria-label^="Collapse "]', '[aria-label^="Expand "]'], headerBar)!
+			.dispatchEvent(new MouseEvent('click', {bubbles: true, altKey: event.altKey}));
```

Comment:
I think this will be enough

```suggestion
		const toggle = $(':scope > button', headerBar)!;

	// The clicked element is either the bar itself or one of its children excluding the button
	if (elementClicked === headerBar || (elementClicked.parentElement === headerBar && elementClicked !== toggle)) {
		toggle.dispatchEvent(new MouseEvent('click', {bubbles: true, altKey: event.altKey}));
```